tools/doc-generator/doc-generator.go

```diff
@@ -184,6 +172,31 @@ func getMetricsNotIncludeInEndpointByDefault() metricList {
 	return metrics
 }
 
+func getVirtControllerMetrics() metricList {
+	err := virt_controller.SetupMetrics(nil, nil, nil, nil, nil, nil, nil, nil)
+	checkError(err)
```

Comment:
`checkError` is really redundant, it doesn't to the coding style IMO, there is no added value in calling a dedicated function for that.

Reply:
it was previously done in this script, do you think I can/makes sense to edit the whole file and remove the `checkError`s?